### PR TITLE
Record bespoke activities

### PR DESCRIPTION
**Pull request by bensagar-ea**
https://eaflood.atlassian.net/browse/WE-1478

### CODE REVIEW

File: src/assets/sass/application.scss [hunk 1]
```diff
@@ -242,3 +242,7 @@ textarea+ .form-hint {
       text-align: center;
     }
 }
+
+.hint{
+  color: grey;
```
Comment (ajaxscape) [inline src/assets/sass/application.scss]: should be: $secondary-text-colour;

File: src/controllers/bespokeOrStandardRules.controller.js [hunk 2]
```diff
@@ -31,7 +30,8 @@ module.exports = class BespokeOrStandardRulesController extends BaseController {
     const isBespoke = request.payload['permit-type'] === 'bespoke'
 
     if (isBespoke) {
-      return this.redirect({ request, h, redirectPath: Routes.BESPOKE_APPLY_OFFLINE.path })
+      // Enter the triage steps for bespoke
+      return this.redirect({ request, h, redirectPath: '/triage/bespoke' })
```
Comment (ajaxscape) [inline src/controllers/bespokeOrStandardRules.controller.js]: Please use the constant for the route.

File: src/models/triage/permitHolderTypeList.model.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  }
+
+  async getFacilityTypeList () {
+    return FacilityTypeList.createList(this.entityContext, this.permitTypesArray, this.permitHolderTypesArray)
+  }
+
+  static async createList (entityContextToUse, permitTypesArray) {
+    // All permit types have the same, full list of permit holder types
+    const permitHolderTypesArray = []
+    PERMIT_HOLDER_TYPE_LIST.forEach((permitHolderType) => permitHolderTypesArray.push(new PermitHolderType(permitHolderType)))
```
Comment (ajaxscape) [inline src/models/triage/permitHolderTypeList.model.js]: Could use map here

**Comment by bensagar-ea** [comment]
Requested changes made

**Pull request merged**